### PR TITLE
daml-script test removal dependency to sandbox-classic

### PR DESCRIPTION
**Pull request by skisel-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: daml-script/test/daml-script-test-runner.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ SANDBOX=$(rlocation $TEST_WORKSPACE/$6)
 
 PORTFILE_DIR=$(mktemp -d)
 PORTFILE=$PORTFILE_DIR/port-file
-$SANDBOX --port=0 --port-file $PORTFILE --static-time $DAR_FILE &
+$SANDBOX --participant=participant-id=example,port=0,port-file=$PORTFILE --stack-traces=true --static-time --enable-user-management=true &
```
Comment (cocreature) [inline daml-script/test/daml-script-test-runner.sh:26]: Why is `enable-user-management=true` not default? We make that default on our production ledgers so I don’t really see why it should be different for SoX.

Comment (skisel-da) [reply daml-script/test/daml-script-test-runner.sh:26]: It's not currently enabled by default:

https://github.com/digital-asset/daml/blob/main/ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/CliConfig.scala#L109

Still I am not sure enabling it should be part of this PR then :)

Comment (cocreature) [reply daml-script/test/daml-script-test-runner.sh:26]: fair, let’s leave it out of this PR. I do think it’s a change worth making though.

**Review by cocreature — APPROVED** [review]
thanks!

**Pull request merged**